Build/prerequisites.md

```diff
@@ -0,0 +1,81 @@
+---
+layout: page
+title: Obtaining and Installing NJOY - Prerequisites
+permalink: /Build/prerequisites.md
+---
+* TOC
+{:toc}
+
+## Prerequisites
+Development for NJOY uses the latest published language standards that are widely supported by compiler vendors, *at the start of development*. Therefore, the minimum language standards are
+                                                                                 
+ - C++14 or higher
+ - Fortran 2003 or higher
+ - [Python 3.4+](https://python.org/downloads/)
+
+Additionally, we use [CMake](https://cmake.org/) to configure the build system and [git](https://git-scm.com) for version control.
+
+ - [CMake 3.2+](https://cmake.org/download/) - Latest Release
+ - [git 2.1+](https://git-scm.com/downloads/)
+
+### On Linux
+ - Use the package manager for your distribution to install the required packages.
+   - Debian distros: `sudo apt install gcc`
+   - Arch: `sudo pacman -Su gcc`
+   - etc.
+ - Ensure that the binaries are in your system path
+   - `echo $PATH`
+   - If not, add it using `export PATH=$PATH:/path/to/binary/directory`
+ - Ensure that the packages have the right version
+   - `[package-name] --version`
+ - If all packages are installed correctly and meet the minimum requirements, you are ready to [build NJOY!](/Build/build.md)
+
+### On Mac
+Why are you using mac?  How does one even mac?  I think the word `brew` is involved?  Whatever.
+
+### On Windows
+Step 1 - Install [Linux](https://www.ubuntu.com/download/desktop).
```

Comment:
While we would all like to have everyone install Linux, this is not the place to make light of it.